internal/commands/push/command.go

```diff
@@ -305,7 +311,13 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 				if err != nil {
 					return err
 				}
-				s.SetMessage(fmt.Sprintf("Installing dependencies: %s...", status.Message))
+
+				message := status.Message
+				if message == "" {
+					message = "finishing up"
```

Comment:
this is a drive-by fix, I noticed at the tail-end of uploading dependencies, we ended up with a spinner message saying `Installing dependencies: ...` I guess `status.Message` gets returned as blank towards the end of our polling there

Reply:
we already have that success message, I like your idea to just `break` early